Obstruction

benhammer opened the pull request:
White house refusal to cooperate with subpoenas

benhammer committed:
White House Subpoenaed

Add the recent House subpoena to Chronology

benhammer committed:
Obstruction

White house refusal to cooperate with subpoenas

JoshData commented:
Let's add this to the charges page too.

And I think we should add a sentence like "The White House's refusal is based on their claim that the House is not affording the President due process rights as he would have in a criminal trial, but an impeachment is not a trial --- its parallel in criminal prosecution is the filing of charges. If the House votes to impeach, a trial would be held in the Senate."

benhammer committed:
Obstruction update

Add sentence about White House letter, add obstruction to list of possible charges
